tracert: add page (#10059)
tldr-pages/tldr: pages/windows/tracert.md

# tracert

> Receive information about each step in the route between your PC and the target.

> > More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/tracert>.

Reviewer: ```suggestion
> More information: <https://learn.microsoft.com/windows-server/administration/windows-commands/tracert>.
```
Just a minor suggestion and the PR is GTG.

> > More information: <https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-xp/bb491018(v=technet.10)?redirectedfrom=MSDN>.

Reviewer: ```suggestion
> More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/tracert>.
```

- Trace a route:

`tracert {{IP}}`

> - Prevent tracert from resolving IP addresses to hostnames:

Reviewer: ```suggestion
- Prevent `tracert` from resolving IP addresses to hostnames:
```

> `tracert -d {{IP}}`

Reviewer: ```suggestion
`tracert /d {{IP}}`
```

> - Force tracert to use IPv4 only:

Reviewer: ```suggestion
- Force `tracert` to use IPv4 only:
```

> `tracert -4 {{IP}}`

Reviewer: ```suggestion
`tracert /4 {{IP}}`
```

> - Force tracert to use IPv6 only:

Reviewer: ```suggestion
- Force `tracert` to use IPv6 only:
```

> `tracert -6 {{IP}}`

Reviewer: ```suggestion
`tracert /6 {{IP}}`
```

> - Show detailed help:
> 
> `tracert /?`
> 
> - Specify the maximum number of hops in the search for the target:
> 
> `tracert -h {{max_hops}} {{IP}}`

Reviewer: ```suggestion
- Specify the maximum number of hops in the search for the target:

`tracert /h {{max_hops}} {{IP}}`

- Display help:

`tracert /?`
```

`Display help` is the generic description we use in most of our pages, also we list it at last.
